lib/sshnp.dart

```diff
@@ -299,13 +302,13 @@ class SSHNP {
     }
   }
 
-
   /// Look up the user name ... we expect a key to have been shared with us by
   /// sshnpd. Let's say we are @human running sshnp, and @daemon is running
   /// sshnpd, then we expect a key to have been shared whose ID is
   /// @human:username.device.sshnp@daemon
   Future<void> fetchRemoteUserName() async {
```

Comment:
Update the documentation here also to explain that this is not called when a remoteUserName is supplied at construction time